web/public/js/controllers/dojos-map-controller.js

```diff
@@ -103,10 +104,17 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
     $state.go('dojo-detail',{country:country, path:path});
   }
 
+  $scope.getDojoURL = function(dojo) {
```

Comment:
You've already set this function from dojoUtils, so you don't need this function definition anymore.